app/assets/javascripts/discourse/app/services/user-tips.js

```diff
@@ -108,4 +126,25 @@ export default class UserTips extends Service {
     this.currentUser.set("user_option.seen_popups", seenUserTips);
     await this.currentUser.save(["seen_popups"]);
   }
+
+  async skipTips() {
+    if (!this.currentUser) {
+      return;
+    }
+
+    this.#availableTips.clear();
+    this.#shouldRenderSet.clear();
+
+    this.currentUser.set("user_option.skip_new_user_tips", true);
+    await this.currentUser.save(["skip_new_user_tips"]);
+  }
+
+  _findObjectInSet(key, value) {
```

Comment:
I do not think this method needs to be that generic.

```suggestion
  _findAvailableTipById(id) {
    return this.#availableTips.values().find((tip) => tip.id === id)
  }
```